Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentSubgraph.php

```diff
@@ -357,6 +415,18 @@ public function jsonSerialize(): array
 
     /** ------------------------------------------- */
 
+    private function findNodeByPathFromStartingNode(NodePath $path, Node $currentNode): ?Node
+    {
+        foreach ($path->getParts() as $edgeName) {
```

Comment:
```suggestion
    private function findNodeByPathFromStartingNode(NodePath $path, Node $startingNode): ?Node
    {
        $currentNode = $startingNode;

        foreach ($path->getParts() as $edgeName) {
```

Nitpicks :roll_eyes: ! I know, I know :sweat_smile: